src/main/java/seedu/address/model/step/Step.java

```diff
@@ -0,0 +1,53 @@
+package seedu.address.model.step;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+public class Step {
```

Comment:
missing javadoc comment for the Step class

Reply:
fixed